contracts/truster/truster.conf

```diff
@@ -0,0 +1,18 @@
+{
+    "files": [
+        "contracts/truster/TrusterLenderPool.sol",
+        "contracts/DamnValuableToken.sol"
+    ],
+    "verify": "TrusterLenderPool:contracts/truster/TrusterLenderPool.spec",
+    // Link a field in the contract to another contract.
+    "link": [
+        "TrusterLenderPool:damnValuableToken=DamnValuableToken"
+    ],
+    "optimistic_loop": true,
+    "server": "production",
+    "prover_version" : "master",
+    "rule_sanity" : "basic",
+
+    "optimistic_contract_recursion": true,
+    "contract_recursion_limit": "4",
```

Comment:
either remove or add currentContract._ to be an option for the dispatcher

Reply:
added currentContract._ to be an option for the dispatcher